### ISSUE TITLE
Gif record button doesn't have a similar hover effect as the other links

### ISSUE THREAD

**Issue by lionleaf** [lionleaf/dwitter#330]
Makes it inconsistent with the rest of the UI:

<img width="129" alt="screen shot 2018-04-22 at 12 28 13 pm" src="https://user-images.githubusercontent.com/610925/39094524-aa43c968-4628-11e8-8c43-b73230e2e65c.png">

**Comment by lionleaf** [comment]
Same with the `delete` button.

**Comment by kipkat** [comment]
```css
.record-text:hover, input.delete-button:hover{
  text-decoration:underline;
}
```

**Comment by lionleaf** [comment]
Closed by #390
